runtime/src/lib.rs

```diff
@@ -320,11 +293,9 @@ impl pallet_balances::Config for Runtime {
     type Event = Event;
     type ExistentialDeposit = ExistentialDeposit;
     type MaxLocks = MaxLocks;
+    type MaxReserves = MaxReserves;
     type WeightInfo = ();
```

Comment:
why isn't this same value as standalone https://github.com/DataHighway-DHX/node/pull/215/files#diff-0ec06ea58bd455f09ce6b3bb4c2c1c0d37bda51c1e1be2151c560c9c973959ecR538

Reply:
I don't know. @festelo  Can you answer this? I haven't changed this.

Reply:
address in #227